### PR TITLE
docs: Add Glitch projects to getting started READMEs

### PR DESCRIPTION
**Pull request by bonniezhou**
Add [Material Starter Kit](https://glitch.com/~material-starter-kit), [Adopt a Pup](https://glitch.com/~adopt-a-pup) and [Shrine](https://glitch.com/~material-example-app-shrine).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,8 @@ Material Components for the web is the successor to [Material Design Lite](https
 
 ### Using NPM
 
+> This guide assumes you have webpack configured to compile Sass into CSS. To configure webpack, please see the full [getting started guide](docs/getting-started.md). If you don't want to walk through the entire guide, check out the [Material Starter Kit](https://glitch.com/~material-starter-kit) for the final result.
```
[the remark this answers is not in the document]
Comment (bonniezhou) [reply README.md]: Done!

File: README.md [hunk 2]
```diff
@@ -95,7 +97,12 @@ This'll initialize text field component on a single `.mdc-text-field` element.
 
 > Please see [quick start demo](https://stackblitz.com/edit/mdc-web-quick-start-demo) on stackblitz for full example.
 
-> Please see [getting started guide](docs/getting-started.md) for pointers on how to configure webpack to bundle JavaScript & CSS.
+
+## Examples using MDC Web
+
+[Shrine](https://glitch.com/~material-example-app-shrine) - An e-commerce app using Text Field, Button, Drawer, List, Image List and Material Theming capabilities
```
[the remark this answers is not in the document]
Comment (bonniezhou) [reply README.md]: How about cross-linking the actual codelabs instead?

The Glitch project will still work but it'll be running the old version, so we'll have to remember to update the Glitch project. This probably merits a larger discussion, but I think  we should have a dedicated person managing all these examples and tooling (DevRel??)

File: README.md [hunk 3]
```diff
@@ -95,7 +97,12 @@ This'll initialize text field component on a single `.mdc-text-field` element.
 
 > Please see [quick start demo](https://stackblitz.com/edit/mdc-web-quick-start-demo) on stackblitz for full example.
 
-> Please see [getting started guide](docs/getting-started.md) for pointers on how to configure webpack to bundle JavaScript & CSS.
+
+## Examples using MDC Web
```
Comment (abhiomkar) [inline README.md]: Just a thought, maybe it makes sense to put this in a separate doc which just shows list of all examples if we're planning to add more later. Idea is to keep quick start guide getting started less overwhelming. WDYT?

Comment (bonniezhou) [reply README.md]: Sorry for the major delay! moved the examples to a separate doc linked from the main README.

File: docs/examples.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<!--docs:
+title: "Getting Started"
+layout: landing
+section: docs
+path: /docs/getting-started/
```
Comment (abhiomkar) [inline docs/examples.md]: `path: /docs/examples/`

File: docs/examples.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+-->
+
+# Examples using MDC Web
+
+[Shrine](https://glitch.com/~material-example-app-shrine) - An e-commerce app using Text Field, Button, Drawer, List, Image List and Material Theming capabilities. You can build Shrine from scratch by walking through the Google Developers Codelabs [MDC-101 Web](https://codelabs.developers.google.com/codelabs/mdc-101-web), [MDC-102 Web](https://codelabs.developers.google.com/codelabs/mdc-102-web), and [MDC-103 Web](https://codelabs.developers.google.com/codelabs/mdc-103-web).
```
Comment (abhiomkar) [inline docs/examples.md]: Add bullet points for each example?

**Pull request merged**